docs/changelogs/CHANGELOG-2.0.md

```diff
@@ -81,5 +87,6 @@
 - Upgrade of Terraform components in issue [#2825](https://github.com/epiphany-platform/epiphany/issues/2825) and [#2853](https://github.com/epiphany-platform/epiphany/issues/2853) will make running re-apply with infrastructure break on existing 1.x clusters. The advice is to deploy a new cluster and migrate data. If needed a manual upgrade path is described [here.](../home/howto/UPGRADE.md#terraform-upgrade-from-epiphany-1.x-to-2.x)
 - Kubernetes container runtime changed. Dockershim and Docker are no longer on Kubernetes hosts.
 - Filebeat `docker` input replaced by `container` input. New field provided for Filebeat as system service installation: `container.id`. Field `kubernetes.container.name` is no longer valid.
+- AWS credentials configuration parameters are renamed from `specification.cloud.credentials.key` and `specification.cloud.credentials.secret` to `specification.cloud.credentials.access_key_id` and `specification.cloud.credentials.secret_access_key`.
```

Comment:
Since this feature is for 2.0.1 shouldn't we move this info under `[2.0.1]` section?

Reply:
Agreed, this willl be 2.0.1 so it can be moved there.

Reply:
Good catch, but is it ok to do such renaming between 2.0.0 and 2.0.1?

Reply:
Yeah in this case I am ok with it as we know that 2.0.0 will not be used in that sense. As I posted yesterday on main 2.0.1 and 2.0.2 will be the versions which are actually gonna be used down the road.